docs/dev-env.md

```diff
@@ -45,6 +44,22 @@ nix (Nix) 2.9.1
 
 The exact version might be different.
 
+### Uninstallation
+
+In case you incorrectly installed Nix or want to remove it, follow these steps:
+
+#### Option 1: Nix Docs Uninstallation
+
+Refer to the official Nix documentation for the recommended uninstallation process : [Official Docs](https://nixos.org/manual/nix/stable/installation/uninstall)
+
+#### Option 2: Brute Force Method
+
+If you encounter difficulties with the standard uninstallation or need to remove Nix forcefully, you can use the following steps:
+1. Use the command `which nix` to locate the Nix binaries.For example, this will give the path to the binaries like /nix/var/nix/profiles/default/bin/nix. For the next step, only consider /nix/var/nix/profiles/default/bin/, i.e., remove 'nix'.
+2. Open the folder containing the Nix files using `xdg-open`.Then search for nix.
+3. Many nix binaries will come for example `nix`, `nix-env`. Use the command `sudo rm nix` and similar commands to delete all files and directories related to Nix (ie that have nix in them).
+4. Once you've removed the Nix files, you can proceed to reinstall Nix using the Determinate Nix Installer mentioned above.
```

Comment:
looks scary and possibly incorrect, you are never removing /nix/store for example.

where did you get these instructions?

Reply:
My mistakes if I explained it incorrectly. Basically the problem I was facing was that even after the official uninstallation it would still show that nix is installed. So I only removed the executables of nix not any data to allow for reinstallation. 
Seems like there is a typo from my part the command is 'xdg-open .', This is for linux use only. And the executables I deleted are below:


![Screenshot from 2024-04-21 16-06-55.png](https://github.com/fedimint/fedimint/assets/147427717/a2839389-9955-419a-8690-bf0676528759)

Reply:
nix/store gets removed by the officail uninstaller maybe but the actual executables does'nt get removed by following it. So I think  brute force step zero would be to execute the official uninstallation and then come to this as a last resort.

Reply:
will defer to @dpc, but I don't like recommending these instructions.